hardhat/tasks/misc.js

```diff
@@ -268,3 +215,122 @@ task("getBalanceOfAccounts", "Get ERC20 or native token balance of account or ad
             }
         }
     });
+task(
+    "misc:vesting-stake-set-event",
+    "get transactions with VestingStakeSet event emitted"
+).setAction(async (taskArgs, hre) => {
+    const {
+        deployments: { deploy, get, log },
+        getNamedAccounts,
+        ethers,
+    } = hre;
+    const staking = await ethers.getContractAt(
+        "IStaking",
+        "0x5684a06CaB22Db16d901fEe2A5C081b4C91eA40e"
+    );
+    //const abi = (await deployments.getArtifact("Staking")).abi;
+    const abi = ["event VestingStakeSet(uint256,uint96)"];
```

Comment:
can we get the ABI from the deployment file instead? to be more reliable in case the event is changing